### DISCUSSION TITLE
What are the risks of delegating `can: *`?

### DISCUSSION THREAD

**Discussion by gobengo** [ucan-wg/spec#153] [General]
Is this something that is ok to do? What are the trade offs?
Should we write tests to ensure it authorizes certain things?
Should using it result in a warning or require the equivalent of `rm` `-f` flag?
How should I decide whether to delegate `*` or `*/*` or neither?

**Comment by bmann** [comment]
@gobengo sorry bengo, I saw this come through but everyone is pretty much slammed in various communications channels.

I'll tag @matheus23 on this directly.
